src/test/java/pages/AddNewCoursePage.java

```diff
@@ -20,12 +25,16 @@ public class AddNewCoursePage extends HomePage{
     public void selectFaculty(String facultyName) {
         selectOption(facultyName,facultyDropdown);
     }
-    public void addNewCourse(String courseName, String facultyName, String courseDescription, String photoLink, String courseStart, String courseEnd, String faculty){
+
+    public List<String> getTitles(){return facultyList.texts();}
+
+    public void addNewCourse(String courseName, String faculty, String courseDescription, String courseStart, String courseEnd){
         addNewCourseTable.shouldBe(Condition.visible);
         courseNameInput.sendKeys(courseName);
-        selectFaculty(faculty);
+        facultyDropdown.click();
+        facultyList.findBy(Condition.exactText(faculty)).click();
         courseDescriptionField.sendKeys(courseDescription);
-        coverPhotoForm.sendKeys("images/qa..jpg");
+        coverPhotoForm.sendKeys("C:\\work\\GraduationProject\\SELENIUM_SELENID\\src\\test\\java\\images\\qa..jpg");
```

Comment:
will be better transfer to variable 
"C:\\work\\GraduationProject\\SELENIUM_SELENID\\src\\test\\java\\images\\qa..jpg"

Reply:
done